### ISSUE TITLE
weirdness with orderBy

### ISSUE THREAD

**Issue by lookfirst** [mikro-orm/mikro-orm#228]
Latest `next`

I don't exactly expect this code to pass, but I also don't expect it to fail this strangely...

Test code:

https://github.com/lookfirst/mikro-example/blob/master/test/db/rig.spec.ts#L27

(note it is `xtest` for now)

```
  console.log node_modules/mikro-orm/dist/utils/Logger.js:19
    [query] begin

  console.log node_modules/mikro-orm/dist/utils/Logger.js:19
    [query] insert into `RigType` (`id`, `name`) values (1, 'rigType 1') [took 1 ms]

  console.log node_modules/mikro-orm/dist/utils/Logger.js:19
    [query] insert into `Rig` (`id`, `type`) values (1, 1) [took 1 ms]

  console.log node_modules/mikro-orm/dist/utils/Logger.js:19
    [query] commit

  console.log node_modules/mikro-orm/dist/utils/Logger.js:19
    [query] select `e0`.* from `Rig` as `e0` order by `e0`.`type` asc [took 1 ms]


Error: select `e0`.* from `RigType` as `e0` where `e0`.`id` in (1) group by `e0`.`id` order by `e0`.`0` asc, `e0`.`1` asc, `e0`.`2` asc - SQLITE_ERROR: no such column: e0.0
Error: SQLITE_ERROR: no such column: e0.0
```

**Comment by B4nan** [comment]
We could sort by PK in this case (that is how it works for FilterQuery), or do you think we should add validation instead?

**Comment by lookfirst** [comment]
I think just by PK is fine since it is effectively unspecified.

**Comment by lookfirst** [comment]
The issue here is that these queries are effectively generic and dynamically built. I'm doing pagination on a table and I want to be able to click the column heading to sort the table by columns. I can just pass the list of sorts `[{columnName: string, direction: string}...]` to the backend without even needing to look at them. So I'd rather that a perfectly valid query, not fail like it is.

**Comment by B4nan** [comment]
Ok I misunderstood it for the first time, I thought the table is joined in the second query so I suggested to order by its PK, but as it is not, I will just ignore the order by in that case. 

Fixed in `3.0.0-alpha.36`

**Comment by lookfirst** [comment]
Confirmed
